shared/dataflow/codeql/dataflow/DataFlow.qll

```diff
@@ -101,6 +101,7 @@ signature module InputSig {
   bindingset[t1, t2]
   predicate compatibleTypes(DataFlowType t1, DataFlowType t2);
 
+  bindingset[t1, t2]
```

Comment:
This relation materialises just fine for Java - are you sure the bindingset is necessary?

Reply:
You are right, not needed after making `typeStrongerThan(t1, t2)` imply `compatibleTypes(t1, t2)`.